src/blog/2019-03-14/blog.json

```diff
@@ -0,0 +1,5 @@
+{
+    "title": "Announcing .NET Framework support with Stryker.NET 0.10",
+    "description": "Stryker.NET 0.10 is here! This brings new features to mutation testing for .NET. Read all about these features in this blog.",
```

Comment:
> This brings new features to mutation testing for .NET. Read all about these features in this blog.

What new features are there? I thought it was mostly full framework support. If that's the case, I would suggest to just put that here.

> This brings mutation testing to the .NET (full) framework for the first time. Read all about it in this blog article.